src/js/components/molecules/PageNation/index.vue

```diff
@@ -0,0 +1,108 @@
+<template lang="html">
+  <ul class="page-wrapper">
+    <li>
+      <app-button
+        class="page-item"
+        :disabled="1 === currentPage ? true : false"
+        @click="handlePageButtonClick(1)"
+      >
+        1
+      </app-button>
+    </li>
+    <li v-if="!(currentPage === 1)">
```

Comment:
[imo]
booleanを反転させずに、下記の方が読みやすいかなと思いました！
```suggestion
    <li v-if="currentPage !== 1">
```

Reply:
修正いたしました！
1b7ce79fd7b1f94592c12a190aba912b315a4456